src/apm.js

```diff
@@ -30,10 +30,6 @@ let apmConfig;
 const isKibanaDistributable = Boolean(build && build.distributable === true);
 
 module.exports = function (serviceName = name) {
-  if (process.env.kbnWorkerType === 'optmzr') {
```

Comment:
Do we want to start APM for both parent and child CLI process? Do they behave differently?

Reply:
I don't know if we want to run APM in both processes, but they do behave differently. The parent process is basically just loading enough of Kibana to understand the config files and proxy requests to the child process which runs the full server and is restarted by the parent process when a server file is changed.

Reply:
Based on the way things are setup it looks like APM is intentionally setup in the parent process, and it uses a different service name to identify itself as the parent.

Reply:
Thanks for the info, yeah makes sense. I am fine with keeping it as it is for now.